drivers/bmx055/bmx055.c

```diff
@@ -234,7 +234,7 @@ int bmx055_gyro_read(const bmx055_t *dev, int16_t *data)
     uint16_t scale;
 
     /* converting scale info into real scaling values */
-    scale = (GYRO_2000_DPS / pow(2, dev->p.gyro_scale));
+    scale = (GYRO_2000_DPS / (1U << dev->p.gyro_scale));
```

Comment:
or as @chrysn pointed out 

```suggestion
    scale = GYRO_2000_DPS >> dev->p.gyro_scale;
```